docs/changelog.rst

```diff
@@ -145,6 +145,7 @@ New features:
   plugin which allows to replace fields based on a given library query.
 * :doc:`/plugins/lyrics`: Add LRCLIB as a new lyrics provider and a new
   `synced` option to prefer synced lyrics over plain lyrics.
+* Sorted the default configuration file :ref: into categories.
```

Comment:
If I don't have an issue that is fixed/implemented I always put the id of the PR implementing it into the changelog. I don't know if that's good practice but I do it. I wanted to fixed your :ref:link link to point to config_default.yaml. I don't know how to do it, do you have an idea? i couldn't find anything in the docs in a quick search where we link to actual files in the code repo. I'm sure we are doing this somewhere / I did this before. 

Anyway, if anything renders wrongly we'll see it in the final docs built from master and could quickfix it there (no time to test locally atm myself)

```suggestion
* Sorted the default configuration file :ref: into categories.
  :bug:`4987`
```


For your problem with rendering docs locally open another issue or discussion or something describing your issues, I'll try to help sort it out :-)